tests/test_utils.py

```diff
@@ -20,3 +21,9 @@ class TestCase(unittest.TestCase):
         [torch.zeros(1, 3, 3, 3), torch.ones(1, 3, 3, 3), torch.full((1, 3, 3, 3), 2)],
         1,
     )
+
+
+def func_req_additional_args(func):
```

Comment:
This function name could sound more boolean. 

e.g. `does_func_have_additional_args()`